src/components/WizardModal/WizardModal.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { usePortal } from '../../hooks/usePortal';
+import { useLockBodyScroll } from '../../hooks/useLockBodyScroll';
+import { getColor, getDepth, getRadii, pxToRem } from '../../utils';
+import { WizardModalSizes } from './WizardModal.enums';
+import { WizardModalProps } from './WizardModal.types';
+import { Padbox } from '../layout';
+
+const widthVariants = {
+  [WizardModalSizes.md]: 600,
+  [WizardModalSizes.lg]: 900,
+};
```

Comment:
We can completely drop the `size` property. This modal will exist only in `lg` variant

Reply:
dropped